# LuxOS: config.h.in z AUR zaplat ma starou hlavicku -> autoheader ho nepregeneruje (chybi PACKAGE/PROG_PATH)
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -39,6 +39,9 @@
 	patch -p0 < cups.patch
 	patch -p0 < id.po.patch
 	patch -p0 < libpng.patch
+	# LuxOS: autoheader (autoconf 2.72+) neprepise config.h.in, jehoz 1. radek nevypada jako od nej
+	# ("Generated automatically from") -> v config.h chybi PACKAGE, PROG_PATH ...; oprava jen hlavicky
+	find . -name config.h.in -exec sed -i '1s/ Generated automatically from / Generated from /' {} +
 }
 
 build() {
